src/debug.ts

```diff
@@ -0,0 +1,301 @@
+    // TODO: this needs review in Seneca core
+    if (trace_node) {
+      trace_node.res = data.res
+      trace_node.err = data.err
+    }
+  }
+
+  if (GLOBALS.EXPRESS_READY) {
+    if (!options.prod) {
+      GLOBALS.WS_SERVER!.emit(GLOBALS.WSPATH!, data);
+    }
+
+    try {
+      data_out = stringify(data)
+
+      var logged = false
+
+      if (GLOBALS.LOG_TO_CONSOLE) {
+        logged = true
+        console.log(data_out)
+      }
+      if (!logged) {
+        for (let u in GLOBALS.LOG_TO_CONSOLE_USERS) {
+          if (
+            GLOBALS.LOG_TO_CONSOLE_USERS[u] &&
+            data.meta.custom.principal.user.handle === u
+          )
+            console.log(data_out)
+        }
+      }
+      data_out = JSON.parse(data_out)
+      data_out.err = data.meta.err
+      data_out.error = data.meta.error
+
+      if (options.prod) {
+        data_out.msg = {}
+        data_out.meta = {
+          id: data.meta.id,
+          start: data.meta.start,
+          end: data.meta.end,
+          pattern: data.meta.pattern,
+          parents: data.meta.parents
+        }
+
+        data_out.err = data.meta.err
+        data_out.error = data.meta.error
+
+        if (data.meta.parents && data.meta.parents.length > 1) {
+          data_out.result_length = JSON.stringify(data_out.res).length
+          data_out.res = null
+          data_out.meta.parents = null
+          data_out.meta.parent = data.meta.parents[0][1]
+        }
+      }
+
+      GLOBALS.WS_SERVER!.clients.forEach((c) => {
+          c.send(JSON.stringify(data_out))
+        });
+      }
+    catch (e) {
+      console.log(e)
+    }
+  }
+}
+
+function debug(this: any, options: any) {
+  const seneca = this;
+  let intern: Intern;
+
+  seneca.ready(() => {
+    if (options.intern_log) {
+      intern = new Intern(seneca);
+      intern.handlePrintTree();
+    }
+  })
+
+  this.outward((ctxt: any, data: any) => {
+    // Dirty fix
+    if (ctxt.data) {
+      outward(seneca, ctxt, options, intern);
+    } else if (data) {
+      outward(seneca, { data }, options, intern);
+    }
+  })
+
+  this.inward((ctxt: any, data: any) => {
+    // Dirty fix
+    if (ctxt.data) {
+      inward(seneca, ctxt, options, intern);
+    } else if (data) {
+      inward(seneca, { data }, options, intern);
+    }
+  })
+
+  return {
+    exports: {
+      native: () => ({})
+    }
+  }
+}
+
+const defaults = {
+  express: {
+    port: 8899,
+    host: 'localhost'
+  },
+  ws: {
+    port: 8898,
+  },
+  intern_log: false,
+  wspath: '/debug',
+  store: false,
+  test: false,
+  prod: false
+}
+
+function bootExpress(options: any) {
+  const { express: expOptions, ws: wsConfig } = options;
+  const app = express();
+
+  app.use(express.static(`${__dirname}/../dist`));
+
+  app.get('/', (req, res) => {
+    res.sendFile('index.html');
+  });
+
+  app.get('/config', (req, res) => {
+    return res.status(200).json({
+      port: wsConfig.port,
+    })
+  });
+
+  app.get('/console', (req, res) => {
```

Comment:
Since logToConsole is now a plugin options, shouldn't we just delete this entire logic, and just log to console when the user sets `logToConsole: true` ?!